main.go

```diff
@@ -18,10 +18,16 @@ var (
 			"version": "v0.1.0",
 		},
 	})
+
 	httpRequestsTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Name: "http_requests_total",
 		Help: "Count of all HTTP requests",
 	}, []string{"code", "method"})
+
+	httpRequestDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name: "http_request_duration_seconds",
+		Help: "Histogram of all HTTP requests duration",
```

Comment:
Nit: I would say something like, as we have that in the type of metric:
```suggestion
		Help: "Duration of all HTTP requests",
```